### PR TITLE
Secret squirrel

### PR DESCRIPTION
**Pull request by adambraimbridge**
Fixes: https://github.com/financial-times-sandbox/tako/issues/9

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,7 +46,9 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "precise-commits && npm run lint"
+      "pre-commit": "secret-squirrel && precise-commits && npm run lint",
+      "commitmsg": "secret-squirrel-commitmsg",
+      "prepush": "npm run test"
```
Comment (adambraimbridge) [inline package.json]: This seems to not actually run tests

File: secret-squirrel.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+module.exports = {
```
Comment (adambraimbridge) [inline secret-squirrel.js]: Todo: either change `secret squirrel` or stop using `precise-commits`

Comment (simonplend) [reply secret-squirrel.js]: So Secret Squirrel generates a file that uses spaces and not tabs? That's different to the [next convention](https://github.com/Financial-Times/n-gage/blob/master/dotfiles/.editorconfig#L8) :scream_cat: 

Although weirdly it looks like Secret Squirrel is [trying to generate this file with tabs](https://github.com/Financial-Times/secret-squirrel/blob/master/lib/init.js#L34) :thinking:

Comment (adambraimbridge) [reply secret-squirrel.js]: It's an odd one

**Comment by adambraimbridge** [comment]
How to re-open?

**Comment by adambraimbridge** [comment]
Ah, I'll fix the comments and make a new PR

**Pull request closed without merging**